notebooks/week3/03.model_serving_feature_lookup.py

```
@@ -0,0 +1,160 @@
+
+# COMMAND ----------
+
+online_table_name = f"{catalog_name}.{schema_name}.loan_features_online"
+spec = OnlineTableSpec(
+    primary_key_columns=["Id"],
+    source_table_full_name=f"{catalog_name}.{schema_name}.loan_features",
+    run_triggered=OnlineTableSpecTriggeredSchedulingPolicy.from_dict({"triggered": "true"}),
+    perform_full_copy=False,
+)
+
+from databricks.sdk.service.catalog import *
```

Ruff (0.8.2)

Module level import not at top of file

(E402)